src/Microsoft.Framework.Runtime/Project.cs

```diff
@@ -188,6 +190,21 @@ internal static Project GetProjectFromStream(Stream stream, string projectName,
                 }
             }
 
+            var fileVersion = Environment.GetEnvironmentVariable("DNX_ASSEMBLY_FILE_VERSION");
+            if (string.IsNullOrWhiteSpace(fileVersion))
+            {
+                fileVersion = project.Version.ToString();
+            }
+
+            try
+            {
+                project.AssemblyFileVersion = new Version(fileVersion);
+            }
+            catch(FormatException ex)
+            {
+                throw new FormatException("The assembly file version is invalid: " + fileVersion, ex);
```

Comment:
Nah. The error message is really bad. It just says that the string has an invalid format but doesn't tell you what the value is.